cryptus-app/scripts/get_attributes_from_collection.ts

```diff
@@ -0,0 +1,280 @@
+import fetch from "node-fetch";
+import fs from "fs";
+import Moralis from "moralis/node.js";
+
+// async function get_price_for_token_ids(
```

Comment:
Est-ce que tu veux garder tout ce qui est en commentaire pour plus tard?

Reply:
nah got rid of it